diff --git a/vendor/phpunit/phpunit/src/Framework/TestCase.php b/vendor/phpunit/phpunit/src/Framework/TestCase.php
index 1223ca0c8..b5725af4c 100644
--- a/vendor/phpunit/phpunit/src/Framework/TestCase.php
+++ b/vendor/phpunit/phpunit/src/Framework/TestCase.php
@@ -475,14 +475,14 @@ public static function tearDownAfterClass(): void
     /**
      * This method is called before each test.
      */
-    protected function setUp(): void
+    protected function setUp()
     {
     }
 
     /**
      * This method is called after each test.
      */
-    protected function tearDown(): void
+    protected function tearDown()
     {
     }
 
